api.js

```diff
@@ -180,6 +188,25 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
+Api.prototype._restartTimer = function () {
+	if (this._timer) {
+		clearTimeout(this._timer);
+	}
+
+	this._timer = setTimeout(this._onTimeout, this.options.timeout);
```

Comment:
There is no need for `_restartTimer` or `_timer`, just use `lodash.debounce` for `onTimeout`.